api/users/user.service.js

```diff
@@ -2,6 +2,7 @@ const pool = require("../../config/database");
 
 module.exports = {
   getUsers: async (user_sub) => {
+    await pool.query(`SELECT set_config('app.sub','${user_sub}',true)`);
     const result = await pool.query(
       `SELECT name, email FROM "user" WHERE sub = $1`,
```

Comment:
After enabling RLS, we don't really need to have this WHERE clause anymore, because now, the query would only return values that has the `sub` as set in line 5.

Reply:
i fixed it sir